### PR TITLE
tweak navbar

### PR DESCRIPTION
**Pull request by michaelkirk-signal**
PTAL @charlesmchen

### CODE REVIEW

File: SignalMessaging/ViewControllers/OWSTableViewController.m [hunk 1]
```diff
@@ -457,9 +457,7 @@ - (void)loadView
     [self.tableView applyScrollViewInsetsFix];
 
     [self.view addSubview:self.tableView];
-    [self.tableView autoPinWidthToSuperview];
-    [self.tableView autoPinToTopLayoutGuideOfViewController:self withInset:0];
-    [self.tableView autoPinEdgeToSuperviewEdge:ALEdgeBottom];
+    [self.tableView autoPinEdgesToSuperviewEdges];
```
Comment (charlesmchen-signal) [inline SignalMessaging/ViewControllers/OWSTableViewController.m]: This seems like something we'll want to test in the 10+ places we use this vc on iOS 9, 10 and 11 and on iPhone X.

File: SignalMessaging/utils/UIUtil.h [hunk 2]
```diff
@@ -20,8 +20,6 @@ typedef void (^completionBlock)(void);
 + (void)applyRoundedBorderToImageView:(UIImageView *)imageView;
 + (void)removeRoundedBorderToImageView:(UIImageView *__strong *)imageView;
 
-+ (completionBlock)modalCompletionBlock;
-+ (void)applyDefaultSystemAppearence;
-+ (void)applySignalAppearence;
++ (void)setupSignalAppearence;
```
Comment (charlesmchen-signal) [inline SignalMessaging/utils/UIUtil.h]: So we _don't_ need to call this in the share extension?

File: SignalMessaging/utils/UIUtil.m [hunk 3]
```diff
@@ -23,42 +24,26 @@ + (void)removeRoundedBorderToImageView:(UIImageView *__strong *)imageView
...
-
-+ (void)applyDefaultSystemAppearence
++ (void)setupSignalAppearence
 {
+    UINavigationBar.appearance.barTintColor = UIColor.ows_navbarBackgroundColor;
+    UINavigationBar.appearance.tintColor = UIColor.ows_navbarIconColor;
+    UIToolbar.appearance.barTintColor = UIColor.ows_navbarBackgroundColor;
+    UIToolbar.appearance.tintColor = UIColor.ows_navbarIconColor;
+
+    UIBarButtonItem.appearance.tintColor = UIColor.ows_navbarIconColor;
```
Comment (charlesmchen-signal) [inline SignalMessaging/utils/UIUtil.m]: So in terms of not needing the default system appearance, we need to test:

* Camera roll / media picker.
* Email composer.
* SMS Message composer.
* Contact editor.
* Document Picker.
* Sharing UI.

Right?

File: SignalMessaging/utils/UIUtil.h (hunk 2 above)
Comment (michaelkirk-signal) [reply SignalMessaging/utils/UIUtil.h]: Right.

File: SignalMessaging/utils/UIUtil.m (hunk 3 above)
Comment (michaelkirk-signal) [reply SignalMessaging/utils/UIUtil.m]: Yup, I'll add this to the test plan.

File: SignalMessaging/ViewControllers/OWSTableViewController.m (hunk 1 above)
Comment (michaelkirk-signal) [reply SignalMessaging/ViewControllers/OWSTableViewController.m]: I'll add this in the test plan.

**Pull request merged**